src/deluge/gui/ui/sound_editor.cpp

```diff
@@ -1323,6 +1327,10 @@ bool SoundEditor::inSettingsMenu() {
 	return (menuItemNavigationRecord[0] == &settingsRootMenu);
 }
 
+bool SoundEditor::inMidiFollowSubmenu() {
+	return ((getRootUI() == &midiSessionView) && (menuItemNavigationRecord[0] == &midiFollowSubmenu));
+}
```

Comment:
might not need to check for root UI here, I think if menuItem record 0 is midiFollowSubmenu then it means it's the soundEditor root menu